src/parser.djs

```diff
@@ -1516,8 +1529,8 @@ such parseFunctionDeclaration much content isClassStatic
                 throw err
             wow
 
-            very newlineStart is content giv content dose startsWith with '\n'
-            rly newlineStart
+            very textContent is content giv content
+            rly plz startsWithNewLine with textContent thx
```

Comment:
```
$ cat snippet.djs
rly plz startsWithNewLine with textContent thx
wow

$ dogescript snippet.djs  --beautify
if (startsWithNewLine(textContent)) {}
```

you can invoke function calls by making use of  `thx` to terminate them in certain spots. However if you'd like me to keep the syntax consistent or if it helps to decipher by just doing newLineStart, I don't mind changing that.

Reply:
at the time I wrote this code I don't think that worked but it was probably fixed in 2.4

looks fine